timescaledb/how-to-guides/compression/manually-compress-chunks.md

```diff
@@ -86,3 +86,29 @@ SELECT compress_chunk(i, if_not_compressed => true)
         now()::timestamp - INTERVAL '3 weeks'
     ) i;
 ```
+
+## Roll up uncompressed chunks when compressing
+
+In TimescaleDB 2.9 and later, you can roll up multiple uncompressed chunks into
+a previously compressed chunk as part of your compression procedure. This allows
+you to have uncompressed chunks with a smaller chunk time interval than the
+compressed chunks. For example, if you have multiple smaller uncompressed chunks
+in your data, you can roll them up into a single compressed chunk. This allows
+you to have much smaller uncompressed chunk intervals, which reduces your disk
+space use for uncompressed data.
```

Comment:
I find the two `This allows...` sentences repetitive in a confusing way. IMO the second is better because it explains the benefit, consider deleting the first or collapsing it into the second?